### ISSUE TITLE
Fail to launch it on a Falcon Heavy

### ISSUE THREAD

**Issue by seoriy** [kelseyhightower/nocode#1235]

**Comment by alitriki** [comment]
Try debug mode and make sure to echo "Don't panic !" on the Tesla screen.
PS: also make sure 
`Tesla.color==Color(Destination.planet)`

**Comment by jarlefosen** [comment]
Center core successfully ran this sequence on descent 🚀 💥

**Comment by cozyplanes** [comment]
@alitriki Adding code is prohibited by the U.N. laws

**Comment by alitriki** [comment]
@cozyplanes You're absolutely right, make sure your code only executes once Falcon Heavy has left earth's atmosphere. That would make it under U.P. (United Planets) laws where there's no code prohibition
